### ISSUE TITLE
CamelCase in spec vs under_scores in examples

### ISSUE THREAD

**Issue by kparal** [coreos/fcct#15]
The [configuration spec](https://github.com/coreos/fcct/blob/418bebaf8b68cc5a1b0ceab309855fd2ed054381/docs/configuration-v1_0.md) shows all supported keys in CamelCase format, but the [examples document](https://github.com/coreos/fcct/blob/0413321bb05ea40b8358581eee7708d1c8aa371f/docs/examples.md) uses underscores. For example, the examples show:
```
ssh_authorized_keys
home_dir
no_create_home
```
and the spec shows:
```
sshAuthorizedKeys
homeDir
noCreateHome
```

It seems that the examples-style naming is the correct one, because fcct fails when converting an input file with e.g. `sshAuthorizedKeys` in it.

This is very confusing. Can you make spec compliant with the implementation or vice versa? :-)

**Comment by ajeddeloh** [comment]
Good catch! https://github.com/coreos/fcct/pull/25 should fix this

**Comment by ajeddeloh** [comment]
closed via #25
